src/ostorlab/runtimes/local/models/models.py

```diff
@@ -5,8 +5,8 @@
 import json
 import logging
 import pathlib
-from typing import Any, Dict, List, Optional
 import types
+from typing import Any, Dict, List, Optional
```

Comment:
```suggestion
from typing import Any
```

Reply:
Python 3.9 support. We can't  switch yet.